client/landing/gutenboarding/index.tsx

```diff
@@ -94,31 +95,23 @@ window.AppBoot = async () => {
 	// Add accessible-focus listener.
 	accessibleFocus();
 
-	let i18nLocaleData;
+	let initialLocaleData;
 	try {
-		const [ userLocale, { translatedChunks, ...localeData } ]: (
-			| string
-			| any
-		 )[] = await getLocale();
-		i18nLocaleData = localeData;
-		setLocaleData( localeData );
```

Comment:
I removed setLocaleData from here because it is called in https://github.com/WordPress/gutenberg/blob/master/packages/i18n/src/create-i18n.js#L189 via the `<I18nProvider>`https://github.com/Automattic/wp-calypso/blob/master/packages/react-i18n/src/index.tsx#L72